hazelcast-client-new/src/main/java/com/hazelcast/client/config/XmlClientConfigBuilder.java

```diff
@@ -132,6 +133,7 @@ protected Document parse(InputStream inputStream) throws Exception {
         DocumentBuilderFactory dbf = DocumentBuilderFactory.newInstance();
         dbf.setNamespaceAware(true);
         final DocumentBuilder builder = dbf.newDocumentBuilder();
+        Locale.setDefault(Locale.getDefault());
```

Comment:
Changing the default Locale for the whole JVM doesn't seem like a very friendly solution for anyone using Hazelcast.